hw/ip/otbn/dv/otbnsim/sim/state.py

```diff
@@ -18,7 +18,6 @@
 from .trace import Trace, TracePC
 from .wsr import WSRFile
 
-
```

Comment:
Doesn't dropping this line cause a flake8 lint error?

Reply:
There was no lint error. But I've added the line back and re-pushed.